src/components/MessagesList/MessagesGroup/Message/MessageButtonsBar/MessageButtonsBar.vue

```diff
@@ -599,6 +604,15 @@ export default {
 			this.$router.push({ name: 'conversation', params: { token: conversation.token } }).catch(err => console.debug(`Error while pushing the new conversation's route: ${err}`))
 		},
 
+		async handleCopyMessageText() {
+			try {
+				await navigator.clipboard.writeText(this.messageObject.message)
```

Comment:
Btw this does not replace mentions, right? I think it should copy `@admin` in that case, not the placeholder.